assets/css/sensei-course-theme/quiz.scss

```diff
@@ -104,6 +104,62 @@ $tablet-breakpoint: 768px;
 	}
 }
 
+.sensei-course-theme.quiz {
+	form {
+		#sensei-quiz-list {
+			input[type=radio],
```

Comment:
What do you think about adding custom classes to these elements so that we can target them by class name as opposed to these somewhat cryptic element selectors?

Reply:
That's a good idea and I also prefer using selectors this way. I've updated it here https://github.com/Automattic/sensei/pull/7124/commits/c23e6c79aa499a3641ec18dacd837904657f91fd